contracts/JBveBanny.sol

```diff
@@ -78,6 +88,12 @@ contract JBveBanny is ERC721Votes, ERC721Enumerable, Ownable, ReentrancyGuard, J
   */
   uint256 public immutable projectId;
 
+  /** 
+    @notice 
+    The options for lock durations.
+  */
+  uint256[] public lockDurationOptions;
```

Comment:
@mejango should we maybe use an enum instead of an array unless we don't want the duration options to be fixed

Reply:
the idea here is that the duration options are not fixed. They are specifiable by the contract deployer.

Reply:
got it I asked only since earlier it was fixed